bazel/BUILD

```diff
@@ -17,15 +17,24 @@ licenses(["notice"])  # Apache v2
 package(default_visibility = ["//:__subpackages__"])
 
 load(":cc_grpc_library.bzl", "cc_grpc_library")
+load(":compilation_database.bzl", "compilation_database")
 
 proto_library(
     name = "well_known_protos_list",
     srcs = ["@com_google_protobuf//:well_known_protos"],
 )
 
-cc_grpc_library(
-    name = "well_known_protos",
-    srcs = "well_known_protos_list",
-    proto_only = True,
-    deps = [],
+#cc_grpc_library(
```

Comment:
is this comment an unintentional leftover?

Reply:
I think @ctiller is merely trying things out; we got automatically assigned because of owners :)

Reply:
the rule is broken at head... so I'm not sure

Reply:
Consider this a proof of implementability that'll be handed off to someone to do build system refactoring in the future.